One of the Python lint workflows was still failing. Not sure if you have access to the error, so I copied it for reference:

```
env["RUSTFLAGS"] = " ".join(rust_flags)
     cargo_command = ["cargo", "build", "--release", f"--target={props.arch}"]
     self.checked_command(cargo_command, cwd="bootloader", env=env)
-    binary_path = os.path.join(
-      CARGO_TARGET_DIR, props.arch, "release", "bootloader"
-    )
+    binary_path = os.path.join(CARGO_TARGET_DIR, props.arch, "release",
+                               "bootloader")
     objcopy_command = [
         "llvm-objcopy", "-O", "binary", binary_path, f"{binary_path}.bin"
     ]
@@ -816,8 +8[15](https://github.com/google/OpenSK/actions/runs/3297670137/jobs/5438836688#step:5:16),7 @@
       if self.args.application:
         self.install_padding()
         self.install_tab_file(
-          f"{CARGO_TARGET_DIR}/tab/{self.args.application}.tab"
-        )
+            f"{CARGO_TARGET_DIR}/tab/{self.args.application}.tab")
         self.install_metadata()
         if not self.verify_flashed_app(self.args.application):
           error(("It seems that something went wrong. App/example not found "
Error: Process completed with exit code 1.
```

--- a/deploy.py
+++ b/deploy.py
@@ -162,6 +162,7 @@ APP_HEAP_SIZE = 90000
 
 CARGO_TARGET_DIR = os.environ.get("CARGO_TARGET_DIR", "target")
 
+
 def get_supported_boards() -> Tuple[str]:
   """Returns a tuple all valid supported boards."""
   boards = []
@@ -402,9 +403,8 @@ class OpenSKInstaller:
     env["RUSTFLAGS"] = " ".join(rust_flags)
     cargo_command = ["cargo", "build", "--release", f"--target={props.arch}"]
     self.checked_command(cargo_command, cwd="bootloader", env=env)
-    binary_path = os.path.join(
-      CARGO_TARGET_DIR, props.arch, "release", "bootloader"
-    )
+    binary_path = os.path.join(CARGO_TARGET_DIR, props.arch, "release",
+                               "bootloader")
     objcopy_command = [
         "llvm-objcopy", "-O", "binary", binary_path, f"{binary_path}.bin"
     ]
@@ -814,8 +814,7 @@ class OpenSKInstaller:
       if self.args.application:
         self.install_padding()
         self.install_tab_file(
-          f"{CARGO_TARGET_DIR}/tab/{self.args.application}.tab"
-        )
+            f"{CARGO_TARGET_DIR}/tab/{self.args.application}.tab")
         self.install_metadata()
         if not self.verify_flashed_app(self.args.application):
           error(("It seems that something went wrong. App/example not found "

